app/Http/Services/Tabbar/TabbarService.php

```diff
@@ -70,14 +76,15 @@ private function _getHomeTabInfo($homeParam, $homeTabUrlInfo) {
     }
 
     /**
-     * Contentsタブの情報を取得する
+     * Contentsタブの情報を取得する.
      * 
-     * @param string $contentsParam リクエストパラメータのcontentsの値
-     * @param array $contentsTabUrlInfo Contentsタブ内に表示するページのURL一覧
+     * @param string $contentsParam      リクエストパラメータのcontentsの値
+     * @param array  $contentsTabUrlInfo Contentsタブ内に表示するページのURL一覧
+     *
      * @return string Contentsタブに表示するページのURL
-     * 
      */
-    private function _getContentsTabInfo($contentsParam, $contentsTabUrlInfo) {
+    private function _getContentsTabInfo($contentsParam, $contentsTabUrlInfo)
```

Comment:
__[code_sniffer]__
* Method name "_getContentsTabInfo" should not be prefixed with an underscore to indicate visibility